net/wifischedule/net/usr/bin/wifi_schedule.sh

```diff
@@ -277,14 +277,17 @@ _should_wifi_enabled()
 startup()
 {
     _log "startup"
-    local _enable_wifi=$(_should_wifi_enabled)
-    if [[ ${_enable_wifi} -eq 0 ]]
-    then
-        _log "enable wifi"
-        enable_wifi
-    else 
-        _log "disable wifi"
-        disable_wifi
+    local global_enabled=$(_get_uci_value ${GLOBAL}.enabled) || _exit 1
+    if [ ${global_enabled} -eq 1 ]; then
+        local _enable_wifi=$(_should_wifi_enabled)
+        if [[ ${_enable_wifi} -eq 0 ]]
```

Comment:
[[]] is a bashism. Might want to run through shellcheck in any case.

Reply:
Thanks. It's in the original code, but I can get rid of it anyway.